src/hoc/withAuthRedirect.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Redirect } from "react-router-dom";
+import { connect } from "react-redux";
+
+import {ROUTES} from "./../core/constants"
+
+let mapStateToPropsForRedirect = (state) => ({
```

Comment:
Don't forget to use "const" instead of "let" for functions.
If they change in the future, you will change "const" to "let".

Reply:
sorry. Done